pkg/cmd/server/origin/master.go

```diff
@@ -488,18 +488,12 @@ func (c *MasterConfig) ensureComponentAuthorizationRules() {
 // TODO Have MasterConfig take a fully formed Authorizer
 func (c *MasterConfig) authorizationFilter(handler http.Handler) http.Handler {
 	authorizationEtcd := authorizationetcd.New(c.EtcdHelper)
-	authorizationAttributeBuilder := authorizer.NewAuthorizationAttributeBuilder(c.getRequestsToUsers())
+
+	authorizationAttributeBuilder := authorizer.NewAuthorizationAttributeBuilder(c.getRequestsToUsers(), &authorizer.APIRequestInfoResolver{util.NewStringSet("api", "osapi"), latest.RESTMapper})
 	authz := authorizer.NewAuthorizer(c.MasterAuthorizationNamespace, authorizationEtcd, authorizationEtcd)
 
 	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
 		attributes, err := authorizationAttributeBuilder.GetAttributes(req)
-		// TODO: this significantly relaxes the authorization guarantees - however the unprotected resources need
```

Comment:
Yeah, we need to prove that all URLs covered by authorizationFilter can be unauthenticated.  Also, this really needs to be worked into the definition of the resources in api_installer (rather than orthogonal).